e2e/app/page/data-page.ts

```diff
@@ -21,8 +24,11 @@ export default class DataPage extends AuthenticatedPage {
 
   async isLoaded(): Promise<boolean> {
     try {
-      await this.waitUntilTitleMatch('Data Page');
-      await this.page.waitForXPath(TAB_SELECTOR.showAllTab, {visible: true});
+      await Promise.all([
```

Comment:
This is a bit of a behavior change, where it will do all of them in parallel now instead of sequence. Is that okay? (I don't know if the pages need to load before you can get the tab selectors?)

It looks fine though, thinking about it conceptually, and this also seems better from a code standpoint! Nice job!